playbooks/openshift/deploy_aosi.yml

```diff
@@ -0,0 +1,18 @@
+# #### Dependencies
+#
+# - setup.yml
+# - repo_from_source.yml
+#
+# #### Install ose from built rpms
+# ## defined in install_ose.yml ##
+#
+# - drop yum repo config onto all nodes
+# - yum install packages
+# - configure and run atomic-openshift-installer
+
+---
+- hosts: all
+  sudo: '{{ sudo | default("no") }}'
```

Comment:
Oh, I should also mention, these scripts are using version 1.9.6. So leaving it as sudo until CentOS updates to 2.x.